components/blitz/resources/omero/FS.ice

```diff
@@ -398,6 +399,7 @@ module omero {
                                 int blockSize,
                                 bool ignoreSysFiles,
                                 bool ignoreDirEvents,
+                                bool platformCheck,
```

Comment:
Thanks @mtbc I'll update that card if this ends up in. I guess the alternative @joshmoore is to not backport?